xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2109,6 +2109,9 @@ platform.core.rightsManagement.editRightsForSpace=Editing access rights for spac
 platform.core.rightsManagement.ajaxFailure=An error occurred while communicating with the server. Please check that the server is accessible, and you have the proper rights to perform the requested action.
 platform.core.rightsManagement.saveFailure=An exception occurred while trying to save the current modifications. Please check if you have the proper rights to perform these modifications.
 platform.core.rightsManagement.saveComment={0} {1} right for {2}
+platform.core.rightsManagement.allowed=Allowed
+platform.core.rightsManagement.denied=Denied
+platform.core.rightsManagement.inherit=Inherit
```

Comment:
This is more complex than inherit. For example, when you give one group view right on a document, the view right is denied for all other groups, the value of the parent has no influence here. It is just not set I would say.

Reply:
wdyt of `undefined` for the "not set values"?